src/AssemblerAlign.cpp

```diff
@@ -333,9 +333,9 @@ void Assembler::computeAlignments(
 
     cout << timestamp;
     if (data.storeAlignments) {
-        cout << "Stored compressed alignments." << endl;
+        cout << "Cached compressed alignments for potential reuse." << endl;
```

Comment:
I prefer "storing" instead of "caching" here. "Caching" gives the impression of something temporary and dynamic, which is not the case here.